### PR TITLE
v1.0 release

### PR DESCRIPTION
**Pull request by philwareublox**

### CODE REVIEW

**Comment by philwareublox** [comment]
After the review the version string will be changed to v1.0, MQTT-Anywhere example built and quickly tested before the merge

**Comment by Chatzidimitriou** [comment]
**Summary:**
The repo and the code seems fine. Some minor suggestions for the Readmes, which are not important. Had issues with MQTT-Flex. Maybe we can test MQTT-Flex again to be sure that it works ok?

**More Details below...**

Here I have some suggestions for the cellular tracker Readme:
- In the application remote commands section, maybe we can add an example command (as there is in the main Readme of the repo). It may help the user.
- In the "building the application" section, maybe we can add a link to Peter's instructions if someone wants to build, using VS Code: https://github.com/u-blox/ubxlib_examples_xplr_iot#using-the-nrfconnect-visual-studio-code-extension
- In the "building the application" section, I think we should add a recommended/minimum nRF SDK version (e.g. v2.1.0)

Applications folder Readme:
- There is a Led indication section which describes the Led status. When the cellular application tries to connect, there is a solid yellow LED state, which is not described there. Maybe we should add it?

In terms of code styling etc. I think the repo is in good shape.

In terms of usage, I have not tested everything, but here are my observations:
**MQTT Now:**
- MQTT Now works fine using Thingstream IOT SIM. It sends location and signal quality ok
- I cannot send commands from Thingstream platform to the XPLR-IOT-1 using MQTT Now. The broker disconnects. Maybe its a Thingstream issue? Should we mention it in the Readme?
- Upon reconnection the XPLR-IOT-1 does no longer subscribe to the remote commands topics

**MQTT Anywhere:**
- Can connect, send messages, receive remote commands properly. Works great

**MQTT Flex:**
- Tried that with a 1nce SIM card over NB-IOT in Greece. The .pem and .key files were saved in the SARA module and APN was changed. The filenames and ClientID for Flex has been entered in the mqtt_credentials file.
- The device could connect to the cellular network but had trouble connecting to MQTT-SN Thingstream broker. (had tried the same certification files and credentials with Sensor Aggregation and it did succeed to connect). Could not get MQTT-Flex to connect. There was always a timeout error (ubxlib -9 error code).
- At one point, after the failed attempt to connect, the # 1 button was unresponsive and the device seemed as hanged. (however, that most probably was a one-time issue - i just mention it here).

File: applications/common/log.c [hunk 1]
```diff
@@ -32,8 +32,8 @@
  * -------------------------------------------------------------- */
 
 /// DO NOT PUT printLog() INSIDE this MUTEX LOCK!!!
-#define MUTEX_LOCK if (pLogMutex != NULL) uPortMutexLock(pLogMutex)
-#define MUTEX_UNLOCK if (pLogMutex != NULL) uPortMutexUnlock(pLogMutex)
+#define MUTEX_LOCK if (pLogMutex != NULL) uPortMutexLock(pLogMutex); {
+#define MUTEX_UNLOCK } if (pLogMutex != NULL) uPortMutexUnlock(pLogMutex);
```
Comment (rebatoma) [inline applications/common/log.c:36]: are the parentheses here OK?

**Review by rebatoma — APPROVED** [review]
LGTM

**Comment by philwareublox** [comment]
Yes. I've done this so that anyone using the MUTEXLOCK marco can't exit the function without balancing the brackets, and remembering to unlock the mutex.
Rob does the same thing in his ubxlib.

**Pull request merged**